doc/services/device_mgmt/smp_groups/smp_group_8.rst

```diff
@@ -445,3 +447,80 @@ where:
     +-----------------------+---------------------------------------------------+
 
 In case when "rc" is not 0, success, the other fields will not appear.
+
+Supported file hash/checksum types
+**********************************
+
+Command allows listing which hash and checksum types are available on a device.
+Requires Kconfig :kconfig:option:`CONFIG_FS_MGMT_CHECKSUM_HASH_SUPPORTED_CMD`
```

Comment:
Use the `CONFIG_MCUMGR_GRP_FS_CHECKSUM_HASH_SUPPORTED_CMD`, to keep the new Kconfig option format for MCUmgr, as You have already did here https://github.com/zephyrproject-rtos/zephyr/pull/50578/files#diff-834286adf361cd54c1b8c09197543580009a48bd6988bc78240a2487ba731f0eR91

Reply:
Fixed